src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/enchanting/AutoEnchanter.java

```diff
@@ -117,7 +116,7 @@ protected MachineRecipe enchant(BlockMenu menu, ItemStack target, ItemStack ench
 
     private boolean isEnchantable(@Nullable ItemStack item) {
         // stops endless checks of getByItem for enchanted book stacks.
-        if (item != null && !item.getType().isAir() && item.getType() != Material.ENCHANTED_BOOK) {
+        if (item != null && !item.getType().isAir() && item.getType() != Material.ENCHANTED_BOOK && !hasIgnoredLore(item)) {
```

Comment:
I'd flip the order of these material checks to just make this a little quicker
```suggestion
        if (item != null && item.getType() != Material.ENCHANTED_BOOK && !item.getType().isAir() && !hasIgnoredLore(item)) {
```